daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/archive/DecodeV1Spec.scala

```diff
@@ -999,6 +1000,60 @@ class DecodeV1Spec
       }
     }
 
+    s"Reject interface definitions in modules iff version < ${LV.Features.interfaces}" in {
+      val i = DamlLf1.DefInterface.newBuilder()
+      i.setTyconInternedDname(1)
+      i.setParamInternedStr(0)
+      i.setPrecond(unitExpr)
+
+      val m = DamlLf1.Module.newBuilder()
+      m.setNameInternedDname(0)
+      m.addInterfaces(i.build())
```

Comment:
Is there any reason you delay the call to `build()` here ? 
Note the protobuf method such as `.addInterfaces` are overiding with the message and the builder for this message, i.e. your `.build()` is not needed here.